sdk/lib/3rdparty/freetype/src/smooth/ftsmooth.c

```diff
@@ -31,6 +30,26 @@
   static FT_Error
   ft_smooth_init( FT_Renderer  render )
   {
+
+#ifndef FT_CONFIG_OPTION_SUBPIXEL_RENDERING
+
+    FT_Vector*  sub = render->root.library->lcd_geometry;
+
+
+    /* set up default subpixel geometry for striped RGB panels. */
+    sub[0].x = -21;
+    sub[0].y = 0;
+    sub[1].x = 0;
+    sub[1].y = 0;
+    sub[2].x = 21;
+    sub[2].y = 0;
+
+#elif 0   /* or else, once ClearType patents expire */
```

Comment:
This could be re-enabled, see: https://freetype.org/patents.html

Reply:
@katahiromz So, is it possible to enable this code ("elif 0" -> "else"), or does it still cause some problems?

Reply:
This is freetype original line.